Bottom: 55b6551b6e077495a6ea0621cd1fdc44e44e3c20
Top:    1cb48247c7ea5f6dd4bccbc599d930df0f149e09
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 23:54:57 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 1bbb1b5..7e79c30 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2618,7 +2618,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
